lib/spack/spack/version.py

```diff
@@ -146,6 +146,10 @@ def underscored(self):
     def dashed(self):
         return '-'.join(str(x) for x in self.version)
 
+    @property
+    def joined(self):
```

Comment:
although some others are not documented, i guess it does not hurt to document this one :smile: